Luke7787/ImmaculateInventors
# Feature/folders be

Summary:
-adding an item now requires a folder
-deleting an item removes it from the folder and user

How to test:

Try adding an item:
(Post)http://localhost:8000/items/
{
    "name": "bossbaby9",
    "quantity": 1,
    "folder": "66202c1c3b6de273a0997c62"
}
-name is the name of the item, quantity is number of items, folder is the id of the folder you want to add the item to
-check to see that the item id is in the folder's item array and in the user's item array
-try removing the "folder": "id" field in the json body before adding an item, there should be an error

Try deleting an item:
(Delete)http://localhost:8000/items?id=6622cd57be8a650c3d696b1b
-id is the id of the item you want to delete
-check that the item is gone from both the user's item array and folder's item array
